exercism/annalyns-infiltration/LincolnInitialSolution.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+
+static class QuestLogic
+{
+    public static bool CanFastAttack(bool knightIsAwake)
+    {
+        if(knightIsAwake){
+            return false;
+        }
+        else{
+            return true;
+        }
+    }
+
+    public static bool CanSpy(bool knightIsAwake, bool archerIsAwake, bool prisonerIsAwake)
+    {
+        if(knightIsAwake || archerIsAwake || prisonerIsAwake){
+            return true;
+        }
+        else{
+            return false;
+        }
```

Comment:
Think here: what does the `if` statement _already_ evaluate to?

Reply:
didn't realize i could put complex booleans with or/and operators directly into a return statement.